everything starts failing like this because it's looking for `errata.stage.engineering.redhat.com` and the fixtures aren't found

```
___________________________________________________________________________________________________________________ ERROR at setup of TestAddBuilds.test_add_builds_url ____________________________________________________________________________________________________________________

monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7f58245be4a0>, mock_get = <errata_tool.tests.conftest.RequestRecorder object at 0x7f58244c97b0>

    @pytest.fixture
    def advisory(monkeypatch, mock_get):
        monkeypatch.delattr('requests.sessions.Session.request')
        monkeypatch.setattr(ErrataConnector, '_auth', None)
        monkeypatch.setattr(requests, 'get', mock_get)
>       return Erratum(errata_id=33840)
               ^^^^^^^^^^^^^^^^^^^^^^^^

errata_tool/tests/conftest.py:98: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
errata_tool/erratum.py:132: in __init__
    self._fetch(kwargs['errata_id'])
errata_tool/erratum.py:187: in _fetch
    r = self._get(endpoint)
        ^^^^^^^^^^^^^^^^^^^
errata_tool/connector.py:163: in _get
    ret_json = ret_data.json()
               ^^^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <errata_tool.tests.conftest.MockResponse object at 0x7f58244ba090>

    def json(self):
        try:
>           with open(self._fixture) as fp:
                 ^^^^^^^^^^^^^^^^^^^
E           FileNotFoundError: [Errno 2] No such file or directory: 'https://errata.stage.engineering.redhat.com/advisory/33840.json'

errata_tool/tests/conftest.py:50: FileNotFoundError
```

--- a/errata_tool/tests/cli/test_main_cli.py
+++ b/errata_tool/tests/cli/test_main_cli.py
@@ -35,6 +35,7 @@ def test_staging_connector(monkeypatch):
     argv = ['errata-tool', '--stage', 'release', 'get', 'rhceph-2.4']
     monkeypatch.setattr(sys, 'argv', argv)
     monkeypatch.setattr(errata_tool.cli.release, 'get', lambda x: None)
+    monkeypatch.setattr(ErrataConnector, '_url', 'https://fake.com')
     main.main()
     expected = 'https://errata.stage.engineering.redhat.com'
     assert ErrataConnector._url == expected

